content/en/tags-type.md

```diff
@@ -297,3 +57,12 @@ var FOO = 1;
 var FOO = 1;
 ```
 {% endexample %}
+
+To indicate that a variable is of a certain type, you can cast it with @type. The surrounding
```

Comment:
This appears to be a Closure Compiler thing that has nothing to do with JSDoc, so please remove this example.

Reply:
So: we have a bunch of examples that refer to Closure Compiler all around the site, and I strongly think that casting is a very important use of `@type` for Closure. With that in mind, I've updated the example to make it clear it's just for Closure—what do you think now?

To be clear, I'm very happy to be overruled, just stating my case a bit more clearly.